parsers/jscript.c

```diff
@@ -1805,17 +1841,38 @@ static bool parseMethods (tokenInfo *const token, const tokenInfo *const class,
 						}
 						else if (isType (token, TOKEN_OPEN_PAREN))
 						{
-							skipArgumentList (token, false, NULL);
+							vStringDelete (signature);
+							signature = vStringNew ();
```

Comment:
If you follow the 1new1delete approach, you can convert these vStringDelete and vStringNew into vStringClear.

Reply:
like:
```C
   vStringClear (signature);
```